docs/content/troubleshooting.md

```diff
@@ -1,113 +1,119 @@
-[//]: # (Doc of docs reference 23)
 [//]: # (TODO - tech writer pass)
 
 # Troubleshooting
 
-* [Errors](#errors)
-* [Exceptions](#exceptions)
-* [Problems](#problems)
-
-We also maintain a [known issues]({{urlRoot}}/known-issues) page for any open bugs in the SpatialOS GDK for Unity.
+<%(TOC)%>
 
+<%(Callout type="tip" message="We also maintain a [known issues]({{urlRoot}}/known-issues) page for any open bugs in the SpatialOS GDK for Unity.")%>
 
 ## Errors
 
-#### Error reported
-When building workers (Unity Editor menu: **Build For Cloud** > **my_worker_name**):
+#### Error building player because build target was unsupported
+When building workers by selecting **Build For Cloud** > **my_worker_name** the following error may occur:
 **_Error building player because build target was unsupported_**</br>
 
-**Issue and workarounds**<br/>
- You don't have the right Unity build settings enabled.<br/>
+**Cause**<br/>
+ You don't have the correct Unity build settings set.<br/>
```

Comment:
Its not about setting build settings, more about having build support for those platforms installed

Reply:
Clarified this.